nautobot_golden_config/migrations/0005_json_compliance_rule.py

```diff
@@ -22,7 +22,11 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunPython(code=jsonify),
+        migrations.AddField(
```

Comment:
We can not modify anything in the existing migrations - this modification of 0005 should be moved into a new migration.

The reason of that is for existing users that 0005 changes have been already made and Django would not look into this file as it already considers it applied

Reply:
I had to do that for new deployments, it would be useful if someone else can clone the repo and test. The order of operations in that migration did not allow for 25 to work.